Makefile

```diff
@@ -1,9 +1,3 @@
-# Update this variable, then run 'make setver'
-# Naming convention:
-#	for stable releases we use "1.0.0" format
-#   for pre-releases, we use   "1.0.0-beta.2" format
-VERSION=1.2.5
-
```

Comment:
that's why I take it out. I just want to set that on the command line so I don't see any reason on have a fixed or default value there.